### DISCUSSION TITLE
[vote ended on 2023-11-08] Move votes and discussions to the Ansible forum

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#279] [Votes done]
1. We merge https://github.com/ansible-community/community-topics/pull/277
2. From the point of that PR being merged, new community topics should be started in the forum, and all votes held on them and existing topics will happen in the forum. The details are explained in the PR https://github.com/ansible-community/community-topics/pull/277.
3. Existing topics in https://github.com/ansible-community/community-topics/ will stay, unless much more discussion is expected. Then they can be moved to the forum and closed in this repository, with links between the issue and the forum topic.
4. The move of votes and discussions to the forum is under a trial period until December 12th, 2023:
   - By default, we will continue to discuss and vote in the forum.
   - If someone wants to **revert the decision** to move all votes to the forum, they need to create a vote **in community-topics** (not the forum!) between November 29nd and December 12th that the trial period should not be extended.
   - The vote must run for at least one week and at most two weeks, and all Steering Committee members must be pinged on GitHub **and** in the forum, to make sure everyone is aware of the vote.
   - If such a vote exists and ends after December 12th, the trial period will be prolonged until the end of the vote.
   - If the vote succeeds, the decision and the PR https://github.com/ansible-community/community-topics/pull/277 is reverted, and votes and discussions continue in community-topics.
   - If the vote does not succeed, the trial ends successfully and voting and discussion is continued in the forum, with votes and new discussions exclusively in the forum.

Vote with +1 to accept this proposal, and -1 to reject it. If you want to discuss it, please use #273.

**Comment by gotmax23** [comment]
+1

**Comment by acozine** [comment]
Sounds good to me +1.

**Comment by felixfontein** [comment]
+1

**Comment by leogallego** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by gundalow** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by samccann** [comment]
+1

**Comment by russoz** [comment]
+1
